ms2rescore/maxquant.py

```diff
@@ -105,7 +105,7 @@ def _set_mass_error_unit(self) -> None:
         """Get mass error unit from DataFrame columns."""
         if "Mass error [Da]" in self._obj.columns:
             self._mass_error_unit = "Da"
-        elif "Mass Error [ppm]" in self._obj.columns:
+        elif "Mass error [ppm]" in self._obj.columns:
```

Comment:
Problem here is both exist. We need to allow both versions (and potentially also deal with this downstream?)